sdk/src/kalix.ts

```diff
@@ -278,6 +278,14 @@ export interface ServiceMap {
   [key: string]: protobuf.Service;
 }
 
+/** @internal */
+enum UserFunctionErrorSeverity {
+  UNSPECIFIED = 0,
+  ERROR = 1,
+  WARNING = 2,
+  INFO = 3,
+}
```

Comment:
Forgot that the protocol types can only be used in type position. Using an enum compiles in a reference to it, but these types are not packaged in the dist. So mirror the severity enum here for switches. Seems a bit awkward, so maybe the protocol typing and whether that gets included in the package, should be revisited.